roofit/roofitcore/src/RooAbsArg.cxx

```diff
@@ -2051,6 +2055,10 @@ void RooAbsArg::graphVizTree(ostream& os, const char* delimiter, bool useTitle,
     string typeFormat = "\\texttt{";
     string nodeType = (useLatex) ? typeFormat+node->IsA()->GetName()+"}" : node->IsA()->GetName();
 
+    if (auto realNode = dynamic_cast<RooAbsReal*>(node)) {
```

Comment:
Don't know what Guido Van Rossum would say to this :smile: 
https://hub.packtpub.com/why-guido-van-rossum-quit/

But I like it!

Reply:
He'd say that he's not concerned, because it's C++. 🙂